go/vt/vttablet/tabletmanager/rpc_backup.go

```diff
@@ -124,6 +134,16 @@ func (tm *TabletManager) Backup(ctx context.Context, concurrency int, logger log
 			}
 			returnErr = err
 		}
+		// Tell Orchestrator we're no longer stopped on purpose.
```

Comment:
@Zhannan what about here? Should this also be called only if `err == nil`?

Reply:
makes sense!